Patches/JPT Open The Windows/Openvindows.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+	<Operation Class="PatchOperationFindMod">
+		<mods>
+		<li>[JPT] Open The Windows</li>
+		</mods>
+		
+		<match Class="PatchOperationSequence">
+			<success></success>
+			<operations>
+
+		<!-- -->
+      		<li Class="PatchOperationReplace">
+        		<xpath>Defs/ThingDef[@Name = "WindowBase"]/fillPercent</xpath>
+        		<value>
+    			<fillPercent>0.6</fillPercent>

```

Comment:
This does mean that windows can be fired through whether they are open or closed--I don't know if you consider this to be the desired behavior or not.

Reply:
I've actually yet to test open/window, but the alternative to have permanently shut window which behaves exactly like a wall, which is way worse.
Plus, I doubt a panel of glass is going to stop anyone from shooting through it.